components/KitBackgroundScreen.js

```diff
@@ -16,7 +16,7 @@ export default function KitBackgroundScreen(props) {
       backgroundColor: Colors.KIT_WHITE
     },
     container: {
-      padding: 16,
+      padding: props.paddinglol || 16,
```

Comment:
Minor, but not sure what this variable name means.

Not a blocking change

Reply:
ohh that's like optional padding but I can remove it if need be

Reply:
i'll change the variable name to padding